Code/Tools/ProjectManager/Source/GemCatalog/GemFilterWidget.cpp

```diff
@@ -253,48 +260,52 @@ namespace O3DE::ProjectManager
         m_statusFilter->deleteLater();
         m_statusFilter = filterWidget;
 
-        const GemSortFilterProxyModel::GemStatus currentFilterState = m_filterProxyModel->GetGemStatus();
+        const GemSortFilterProxyModel::GemSelected currentFilterState = m_filterProxyModel->GetGemSelected();
         const QList<QAbstractButton*> buttons = m_statusFilter->GetButtonGroup()->buttons();
-        for (int statusFilterIndex = 0; statusFilterIndex < buttons.size(); ++statusFilterIndex)
-        {
-            const GemSortFilterProxyModel::GemStatus gemStatus = static_cast<GemSortFilterProxyModel::GemStatus>(statusFilterIndex);
-            QAbstractButton* button = buttons[statusFilterIndex];
 
-            if (static_cast<GemSortFilterProxyModel::GemStatus>(statusFilterIndex) == currentFilterState)
+        // unselected/selected checkboxes 
+        buttons[0]->setChecked(m_filterProxyModel->GetGemSelected() == GemSortFilterProxyModel::GemSelected::Unselected); 
```

Comment:
Maybe we could use enums to specify index instead of 0 and 1? For better readability on the lower logic. Only problem is the enums are rather long. The later ones are also 2 and 3 so. Your choice just a suggestion.

GemSortFilterProxyModel::GemSelected::Unselected

Reply:
The enums are also strongly typed so I think I'll just use temp variables with nicer names